tests/test_cli_client.py

```
@@ -1102,6 +1102,37 @@ def handler(request):
+                json=[{"id": "10", "name": "Internal", "description": "Internal only"}],
+            )
+
+        client = _make_client(handler)
+        result = client.jira_security_levels("PROJ")
+        assert len(result) == 1
+        assert result[0]["name"] == "Internal"
+
+    def test_jira_security_levels_with_token(self):
+        def handler(request):
+            body = json.loads(request.content)
+            assert body["jira_token"] == "tok"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "jira_token"

(S105)